src/bandersnatch/verify.py

```diff
@@ -0,0 +1,193 @@
+import aiohttp
+import asyncio
+import concurrent.futures
+import json
+import hashlib
+import logging
+import os
+
+from functools import partial
+from pathlib import Path
+from sys import stderr
+from urllib.parse import urlparse
+
+
+logger = logging.getLogger(__name__)
+
+
+def _convert_url_to_path(url):
+    return urlparse(url).path[1:]
+
+
+async def _get_latest_json(json_path, config):  # noqa: E999
+    url_parts = urlparse(config.master)
+    url = "{}://{}/pypa/{}/json".format(
```

Comment:
I get a 404 going to a url with this format.  Shouldn't it be pypi not pypa, for example:
https://pypi.org/pypa/hostlists/json -> 404
https://pypi.org/pypi/hostlists/json -> 200

Reply:
Yeah, lol! PyPA on the mind. I need to get unit tests going.